community/server/pom.xml

```diff
@@ -159,7 +159,7 @@
 
     <dependency>
       <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-classic</artifactId>
+      <artifactId>logback-core</artifactId>
```

Comment:
@jakewins Well, for one thing we should make up our minds, either we use logback or we don't. As it is now it's semi-removed... which is just confusing. Should be dead easy to write our own adapter thing there, right?